ieeg_epilepsy_ecog/sub-ecog01/ses-postimp/sub-ecog01_ses-postimp_scans.tsv

```diff
@@ -1,3 +1,3 @@
 filename	acq_time
 anat/sub-ecog01_ses-postimp_T1w.nii.gz	1900-01-01T00:00:00
-ieeg/sub-ecog01_ses-postimp_task-seizure_run-01_channels.eeg	1900-01-01T00:00:00
+ieeg/sub-ecog01_ses-postimp_task-seizure_run-01_ieeg.eeg	1900-01-01T00:00:00
```

Comment:
tagging @ftadel to make him aware of this fix for his dataset

Reply:
@sappelhoff Thanks, I reported the fix on OpenNeuro (https://openneuro.org/datasets/ds002904/versions/1.0.1) and on the Brainstorm website.